api/api.go

```diff
@@ -42,6 +62,18 @@ func New() *API {
 		lookupAddr:    net.LookupAddr,
 		lookupCountry: func(ip net.IP) (string, error) { return "", nil },
 		ipFromRequest: ipFromRequest,
+		client: &http.Client{
```

Comment:
@martinp: with HTTP it'd also check whether there's no misconfigured proxy or balancer on the given address, TCP check will always give ok if the port is open.

What about having both - `/test/http/:port` and `/test/tcp/:port`?